.coveragerc

```diff
@@ -434,6 +434,7 @@ omit =
     homeassistant/components/mopar/*
     homeassistant/components/mpchc/media_player.py
     homeassistant/components/mpd/media_player.py
+    homeassistant/components/mqtt/remote.py
```

Comment:
All MQTT platforms need to be covered by tests.

Reply:
I added unit tests.